docs/content/v1/usage/script.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Script Tag

```

Comment:
We shoud probably keep using the default English grammar Title Case for titles, not sentence case here (https://www.scribbr.com/academic-writing/title-case/).

Reply:
Didn't know about this! I am updating everywhere according to this then :+1:

Reply:
Just done in [96ef3fe](https://github.com/Polight/lego/pull/67/commits/96ef3fec7c80489a031d1c95bc5f3f92a7d0359d). I have also removed section titles that are redundant with page titles.